runtime/drivers/olap.go

```diff
@@ -72,9 +72,19 @@ type Statement struct {
 	DryRun           bool
 	Priority         int
 	LongRunning      bool
+	OlapQueryCfg     *OlapQueryCfg
```

Comment:
A separate "OLAP query config" type doesn't really make sense here because all the fields in it are already query config keys (args, priority, timeouts, etc.).

So it should be something like this (but with the docstrings added):

```suggestion
	UseCache bool
	PopulateCache bool
```

Reply:
I wanted to have it as struct here because I wanted to both of them to be set but now I think about it, its ok to omit. Changed